velox/type/UnscaledShortDecimal.h

```diff
@@ -54,14 +54,6 @@ struct UnscaledShortDecimal {
     return unscaledValue_ <= other.unscaledValue_;
   }
 
-  bool operator>(const UnscaledShortDecimal& other) const {
```

Comment:
Why are these functions removed? Is this change intentional?

Reply:
These overloading is not required. But will do in a separate PR.